addons/cookoff/functions/fnc_smoke.sqf

```diff
@@ -31,7 +30,7 @@ private _effects = [_smokeBarrel];
 {
     private _position = [0, -2, 0];
 
-    if (_x isNotEqualTo "#noselection") then {
+    if (_x != "#noselection") then {
```

Comment:
That's a totally pointless change, all it does is worsen things.

Reply:
a) #9758 
b) No it doesn't.